metallica.html

```diff
@@ -1,45 +1 @@
-<style>
```

Comment:
Deleting the whole file isn't okey.

Reply:
We need to start from scratch since ALL the bands need same style

Reply:
Style can be changed, no need to delete everything(pictures, link etc.)